samples/runtimeDI/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
I did not use Eclipse for this PR, this file was already there I don't know why it's marked as new